packages/bits/spec/virtual-camera/types.ts

```diff
@@ -13,6 +13,7 @@ export const CAMERA_DEFAULT_SETTINGS: ICameraSettings = {
     currentSuiteName: "",
```

Comment:
This is as high up in the file as I can comment, so I'll put it here. Per the nova style guide, entries for enums like LensType above should be Pascal case instead of all caps.